fala/locale/cy/LC_MESSAGES/django.po

```diff
@@ -74,13 +76,40 @@ msgid ""
 "For any other benefits issue, ask the legal adviser if you can get free "
 "legal advice or if you will have to pay for it."
 msgstr ""
+"Nid yw cymorth cyfreithiol i gael cyngor am fudd-daliadau lles ond ar gael "
+"os ydych yn apelio yn erbyn penderfyniad a wnaed gan y tribiwnlys nawdd "
+"cymdeithasol. Rhaid i’r apêl hon fod yn yr Uwch Dribiwnlys, yn y Llys Apêl "
+"neu’r Goruchaf Lys.\n"
+"\n"
+"Gydag unrhyw fater arall sy’n ymwneud â budd-daliadau, gofynnwch i’r "
+"cynghorydd cyfreithiol a allwch gael cyngor cyfreithiol am ddim ynteu a fydd "
+"rhaid ichi dalu amdano."
 
 #: fala/common/category_messages.py:13
 msgid ""
 "Legal aid for advice about clinical negligence is usually only available if "
 "you have a child that suffered a brain injury during pregnancy, birth or the "
 "first 8 weeks of life."
 msgstr ""
+"Nid yw cymorth cyfreithiol am esgeuluster clinigol fel arfer ond ar gael os "
+"oes gennych blentyn sydd wedi cael anaf i’r ymennydd yn ystod beichiogrwydd, "
+"genedigaeth neu yn ystod 8 wythnos gyntaf ei fywyd."
+
+#: fala/common/states.py:94
+msgid "Northern Ireland"
+msgstr "Gogledd Iwerddon"
+
+#: fala/common/states.py:98
+msgid "the Isle of Man"
+msgstr "Ynys Manaw"
+
+#: fala/common/states.py:102
+msgid "Jersey"
```

Comment:
as they use the same word, is there value in reducing complexity by not translating this region?

Reply:
As it doesn't increase the complexity, I kept it in, as it might confuse someone in the future as to why Jersey & Guernsey were not translated. Saves them a google search. 😊